docs/source/preparation_input_dataset.rst

```diff
@@ -12,6 +15,73 @@ For more details, please refer to
 `its documentation <http://docs.datalad.org/en/stable/generated/man/datalad-create.html>`_,
 and `DataLad Handbook <https://handbook.datalad.org/en/latest/basics/101-101-create.html>`__.
 
+==============================================================================
+Raw (unzipped) BIDS dataset, or zipped BIDS dataset?
+==============================================================================
+
+* If there are zip files, like ``sub-*.zip``,
+  it would be considered as a zipped BIDS derivatives dataset.
+* If there are only unzipped folders, like folder ``sub-*/``,
+  then it is considered as a raw (unzipped) BIDS dataset.
+* If both zipped file ``sub-*.zip`` and unzipped folders ``sub-*`` present,
+  then it is considered as a zipped BIDS derivatives dataset.
+
+    * Therefore, if you have a raw BIDS dataset, please do not include zipped files
+      called ``sub-*.zip`` in this dataset.
+
+When running ``babs-init``, you will see printed messages describing each input BIDS dataset
+was categorized as a raw (unzipped) dataset or a zipped dataset.
+
+.. _requirements_for_zipped_BIDS_derivatives_dataset:
+
+=====================================================
+Requirements for zipped BIDS derivatives dataset
+=====================================================
+There are several requirements for zipped BIDS derivatives dataset:
+
+Note: an input dataset's name is defined when ``babs-init --name``.
+
+-------------------------
+Naming zip files
+-------------------------
+
+* For single-session dataset, the zip filename should follow the pattern of
+  ``sub-*_<name>*.zip``, where ``<name>`` is the name of this input dataset.
+
+    * Here, ``*ses-*`` is allowed in the zip filenames.
+
+* Similarly, for multi-session dataset, the zip filename should follow the pattern of
+  ``sub-*_ses-*_<name>*.zip``
+* In this dataset, for each subject (for single-session data),
+  or each session (in multi-session data),
+  there should only be one zip file whose filename contains input dataset's name.
+
+    * For example, say we have ``sub-01_ses-A_freesurfer-20-2-3.zip``,
+      where ``freesurfer`` will be the input dataset's name.
+      There should not be another zip file with ``freesurfer`` for this session,
+      e.g., ``sub-01_ses-A_freesurfer-xxx.zip``
+
+-------------------------
+Content of the zip files
+-------------------------
+Within the zip file for a specific subject (or session), there should be a folder
+named by this input dataset's name, e.g., folder ``freesurfer``
```

Comment:
This is standard for fmriprep, freesurfer, qsiprep, and xcp, but is this standard for all bids apps? Additionally, users can technically zip their pipeline outputs to a directory of any name.